app/design/adminhtml/Magento/backend/web/css/source/forms/_fields.less

```diff
@@ -148,6 +148,57 @@
             }
         }
     }
+    > .product-options {
+        > .admin__field {
+            border: 0;
+            margin: 0 0 3rem 0;
+            padding: 0;
+            #mix-grid .row();
+    
+            &.admin__field-wide {
+                &:extend(.abs-field-rows all);
+            }
+    
+            > .admin__field-control {
+                #mix-grid .column(@field-control-grid__column, @field-grid__columns);
+                float: none;
+                display: inline-block;
+                margin-bottom: 3rem;
+                margin-left: 27px;
+                &:last-child {
+                    margin-bottom: 0;
+                }
+            }
+    
+            > .admin__field-label {
+                #mix-grid .column(@field-label-grid__column, @field-grid__columns);
+                float: none;
+                display: inline-block;  
+                vertical-align: top;              
+            }
+      
+            &:extend(.abs-field-sizes all);
+    
+            &.admin__field-no-label {
+                > .admin__field-label {
+                    display: none;
+                }
+    
+                > .admin__field-control {
+                    &:extend(.abs-field-no-label all);
+                }
+            }
+    
+            //  Field with empty table and no thead
+            + .admin__field {
```

Comment:
@speedy008 is the next element selector here (`+`) an intentional choice?

Reply:
Hi @sivaschenko 
I find there is no need for that CSS and i have removed it as well.